exporter/metric/cloudmonitoring.go

```diff
@@ -26,7 +26,7 @@ import (
 	export "go.opentelemetry.io/otel/sdk/export/metric"
 	"golang.org/x/oauth2/google"
 
-	monitoring "cloud.google.com/go/monitoring/apiv3"
+	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
```

Comment:
This doesn't seem related to updating the otel dependency. Looks like you just updated all dependencies?

We have automation for specifically updating the SDK dependency: please update the version number in https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/ba72e89a40870af14a0bc190e62174ab3e86335a/Makefile#L136
and run `make update-otel`.

Reply:
Thanks for the review! Just updated the SDK dependency as requested. 

For the cloud monitoring API, I updated it because of failure in CircleCI build (it complained that the API is deprecated).